manifests/charts/base/templates/endpoints.yaml

```diff
@@ -1,19 +1,4 @@
 {{- if .Values.global.remotePilotAddress }}
-  {{- if not .Values.global.istiod.enabled }}
-apiVersion: v1
-kind: Endpoints
-metadata:
-  name: istio-pilot
-  namespace: {{ .Values.global.istioNamespace }}
-subsets:
-- addresses:
-  - ip: {{ .Values.global.remotePilotAddress }}
-  ports:
```

Comment:
please ensure this does not break multicluster within master, or if doc changes are needed please document the changes.